src/Services/MeasurementCalculationService.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class MeasurementCalculationService
+{
+    public const MM = 'mm';
+    public const CM = 'cm';
+    public const IN = 'in';
+    public const FT = 'ft';
+
+    public static function calculateDimension(int $dimension, string $unitOfMeasurement): string
```

Comment:
```suggestion
    public static function convertUnit(int $value, string $unitOfMeasurement): string
```

Naming to make this method more general